src/Features/CSharp/Portable/CodeRefactorings/ConvertToInterpolatedString/InterpolatedStringRewriter.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeRefactorings.ConvertToInterpolatedString
+{
+    internal partial class ConvertToInterpolatedStringRefactoringProvider
+    {
+        private class InterpolatedStringRewriter : CSharpSyntaxRewriter
```

Comment:
Since you're only rewriting InterpolationSyntax, couldn't you call ReplaceNodes rather than creating a rewriter?